src/ucp/core/ucp_request.h

```diff
@@ -152,11 +152,14 @@ struct ucp_request {
 
             };
 
+            struct {
+                ucp_dt_state_t    dt;       /* Position in the send buffer */
```

Comment:
why not just rename "state" to "dt_state"? because logically, all fields in the request are some sort of "state"..

Reply:
I would not say so. Other fields are not changed during progressing of a protocol. If it will be needed we can move them into the state structure.